src/modules/trash/trash.controller.ts

```diff
@@ -206,8 +206,23 @@ export class TrashController {
   @ApiOperation({
     summary: "Deletes all items from user's trash",
   })
-  clearTrash(@UserDecorator() user: User) {
-    this.trashUseCases.clearTrash(user);
```

Comment:
Trash issue where the trash was not emptied but the response from the server is given, which causes 'ghost' trash items to appear and disappear @CandelR @PixoDev just FYI